.github/workflows/pr-ci-build-container-images-cs9.yml

```diff
@@ -0,0 +1,87 @@
+name: "Test build of CentOS Stream 9 based container images"
+on:
+  pull_request:
+    branches:
+      - master
+      - main
```

Comment:
We don't have 'main' branch in this repository

Reply:
Indeed. I've removed it.